### PR TITLE
Travis setup

### PR DESCRIPTION
**Pull request by pmario**
First attempt.

### CODE REVIEW

File: deploy-gh-pages.sh [hunk 1]
```diff
@@ -0,0 +1,52 @@
+#!/bin/bash
+set -e # exit with nonzero exit code if anything fails
+
+
+if [ $TREVIS == "true" ]
```
Comment (Jermolene) [inline deploy-gh-pages.sh]: Typo?

Comment (Jermolene) [inline deploy-gh-pages.sh]: Should it be "TRAVIS" not "TREVIS"?

File: deploy-gh-pages.sh [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+
+echo
+echo "----> npm run stage"
+echo
+
+# run our compile script, discussed above
+npm run stage
+
+echo
+echo "----> go to tmp/jermolen.github.io/"
```
Comment (Jermolene) [inline deploy-gh-pages.sh]: jermolen == typo

**Comment by pmario** [comment]
not used anymore

**Pull request closed without merging**